paper-input-container.html

```diff
@@ -189,6 +189,7 @@
         top: 0;
         right: 0;
         left: 0;
+        width: 100%;
```

Comment:
Firefox seems to need this or it doesn't animate the width and we get the temporary scroll bar.
I think if it doesn't have a value already set it doesn't animate the width property.
Chrome doesn't need this.